client/src/components/GrillLightsControls.js

```diff
@@ -95,20 +120,55 @@ export default function GrillLightsControls(props) {
     setTailLength(tailLengthVal)
   }
 
-  /* not needed now, saving for potential future use
   function updateEntireState(fetchedState) {
     setAnimation(fetchedState.animation)
     setColor({
-      red: fetchedState.red,
-      green: fetchedState.green,
-      blue: fetchedState.blue
+      r: fetchedState.r,
+      g: fetchedState.g,
+      b: fetchedState.b
     })
     setSpeed(fetchedState.speed)
     setDirection(fetchedState.direction)
     setDensity(fetchedState.density)
     setTailLength(fetchedState.tailLength)
   }
-  */
+
+  // get state from server
+  useEffect(() => {    
```

Comment:
Potential refactor: make this effect call an async function and use `await` to avoid the nested promises.

Reply:
will do in a future PR. Can you point me to your favorite tutorial on learning/understanding how async/await work?